module/assets/helper.js

```diff
@@ -0,0 +1,614 @@
+// Dependencies
+const programsHelper = require(MODULES_BASE_PATH + "/programs/helper");
+const solutionsHelper = require(MODULES_BASE_PATH + "/solutions/helper");
+const userExtensionsHelper = require(MODULES_BASE_PATH +
+  "/user-extension/helper");
+const userRolesHelper = require(MODULES_BASE_PATH + "/user-roles/helper");
+const kafkaProducersHelper = require(ROOT_PATH + "/generics/kafka/producers");
+
+module.exports = class AssetsHelper {
+  /**
+   * Get users assets based on Type .
+   * @method
+   * @name fetchAssets
+   * @param {String} queryData -  query for type data.
+   * @param {Object} [bodyData] - request body data data.
+   * @returns {Promise} returns a promise.
+   */
+  static fetchAssets(queryData, bodyData) {
+    return new Promise(async (resolve, reject) => {
+      try {
+        let organizationAssets;
+        switch (queryData) {
+          case "program":
+            organizationAssets =
+              await programsHelper.queryForOrganizationPrograms(
+                bodyData,
+                queryData
+              );
+            break;
+          case "solution":
+            organizationAssets =
+              await solutionsHelper.queryForOrganizationSolutions(
```

Comment:
can you rename queryForOrganizationSolutions to listOrganizationSolutions @praveenKDass  same for Program as well